intera_examples/scripts/set_interaction_options.py

```diff
@@ -24,18 +24,99 @@
 import intera_interface
 from intera_interface import CHECK_VERSION
 
+class SetInteractionOptions(object):
```

Comment:
This class looks very similar to the ConstrainedZeroG class in the previous script. I propose that you make a new file with a class called `InteractionOptionsPublisher` or something like that, and then have both scripts use this class. This reduces code duplication and makes a class that we will likely want to use in other places in the future.